.stylelintrc

```diff
@@ -68,7 +68,7 @@
         "selector-combinator-space-after": "always",
         "selector-combinator-space-before": "always",
         "selector-descendant-combinator-no-non-space": true,
-        "selector-no-id": true,
+        "selector-max-id": 1,
```

Comment:
Can this rule be added as a comment around namespaces, so we still prevent rules with ids from being added?